em_workflows/czi/flow.py

```diff
@@ -148,8 +151,12 @@ def update_file_metadata(file_path: FilePath, callback_with_zarr: Dict) -> Dict:
         file_path=fps, callback_with_zarr=callback_with_zarrs
     )
     callback_with_zarrs = find_thumb_idx(callback=callback_with_zarrs)
+    utils.copy_workdirs.map(fps, wait_for=[callback_with_zarrs])
```

Comment:
I think we should create a task that only copies logs - and use in these instances

Reply:
Fixed in this commit: https://github.com/niaid/image_portal_workflows/pull/381/commits/f288b1625802d58333cdf617fa8fb5787f316f39